batch/batch/driver/instance_pool.py

```diff
@@ -112,6 +113,8 @@ def __init__(self, app, machine_name_prefix):
 
         self.name_instance = {}
 
+        self.task_manager = aiotools.BackgroundTaskManager()
```

Comment:
Should this be the same TaskManager as the one from the app?

Reply:
hmm. I'm not sure how much it matters. Seems a little better to have InstancePool be responsible for cleaning up its own tasks?

Since everything gets shutdown together when the app shuts down, it's mostly a matter of what code we find most easy to understand.